server/routes/handlers/helpers/referenceData/referenceData.test.ts

```diff
@@ -0,0 +1,37 @@
+import { getReferenceDataItemLabel, ReferenceDataCategories } from './referenceData'
+import { RecallResponse } from '../../../../@types/manage-recalls-api/models/RecallResponse'
+
+describe('getReferenceDataItemLabel', () => {
+  it('returns a label for recall lengths', () => {
+    const label = getReferenceDataItemLabel('recallLengths', RecallResponse.recallLength.FOURTEEN_DAYS)
```

Comment:
Some might argue for exhaustive tests of all values - but I'm OK with this selective testing and we're thinking that default labels will move out of this service completely.